docs/GroupDefs.hpp

```diff
@@ -48,6 +48,49 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between frames, such as
+ * the fixed computational frame (grid frame) and the inertial frame. The
+ * time-dependent parameters of the mapping are adjusted by a feedback control
+ * system in order to follow the dynamical evolution of the black holes. In BBH
+ * simulations the map is typically a composition of maps that include
+ * translation, rotation, scaling, shape, etc.
+ * Each map under the governance of the control system has an associated
+ * time-dependent map parameter \f$\lambda(t)\f$ that is a piecewise Nth order
+ * polynomial. At regular time intervals the control system resets the Nth time
+ * derivative of \f$\lambda(t)\f$ to a new constant value, in order to minimize
+ * an error function \f$Q(t)\f$ that is specific to each map. At each reset
+ * time, the Nth derivative of \f$\lambda(t)\f$ is set to a function \f$U(t)\f$,
+ * called the control signal, that is determined by \f$Q(t)\f$ and its time
+ * derivatives and time integral. Note that \f$\lambda(t)\f$, \f$U(t)\f$, and
+ * \f$Q(t)\f$ can be vectors.
+ *
+ * The key components of the control system are:
+ * - FunctionsOfTime: each map has an associated FunctionOfTime which holds the
+ *   map parameter \f$\lambda(t)\f$ and relevant time derivatives.
+ * - ControlError: each map has an associated ControlError which computes
+ *   the error, \f$Q(t)\f$, in the mapping.
+ * - Averager: an averager can be used to average out the noise in the 'raw'
+ *   \f$Q(t)\f$ returned by the ControlError.
+ * - Controller: the map controller computes the control signal \f$U(t)\f$ from
+ *   \f$Q(t)\f$ and its time integral and time derivatives.
+ *   The control is accomplished by setting the Nth derivative of
+ *   \f$\lambda(t)\f$ to \f$U(t)\f$. Two common controllers are PID
+ *   (proportional/integral/derivative)
+ *   \f[ U(t) = a_{0}\int Q(t) dt + a_{1}Q(t) +a_{2}\frac{dQ}{dt} \f]
+ *   or
+ *   PND (proportional/N derivatives)
+ *   \f[ U(t) = \sum_{k=0}^{N} a_{k} \frac{d^kQ}{dt^k} \f]
+ *   The coefficients \f$ a_{k} \f$ in the computation of \f$U(t)\f$ are chosen
+ *   at each time such that the error \f$Q(t)\f$ will be critically damped
+ *   on a timescale of \f$\tau\f$ (the damping time),
+ *   i.e. \f$Q(t) \propto e^{-t/\tau}\f$.
+ * - TimescaleTuner: each map has a TimescaleTuner which dynamically adjusts
+ *   the damping timescale \f$\tau\f$ appropriately to keep the error \f$Q(t)\f$
+ *   within some specified error bounds.
```

Comment:
Note that the reset time interval should be a fraction of the damping timescale (empirically we have found less than ...)